packages/okta-react-native/ios/ReactNativeOktaSdkBridge.m

```diff
@@ -18,7 +18,12 @@ @interface RCT_EXTERN_MODULE(OktaSdkBridge, RCTEventEmitter)
 
 RCT_EXTERN_METHOD(signIn)
 
-RCT_EXTERN_METHOD(authenticate:(NSString *)sessionToken)
+RCT_EXTERN_METHOD(
```

Comment:
Do we really need this breaking change?

Reply:
As bridge call is always asynced, and this change only introduce return promise to the bridge method, I don't really think it's a breaking change from JS API perspective.

Reply:
I see, it is just additional capability